tests/auto/keras/layers/test_recurrent.py

```diff
@@ -42,6 +42,9 @@ def test_gru(self):
     def test_lstm(self):
         _runner(recurrent.LSTM)
 
+    def test_ntm(self):
```

Comment:
@fchollet you can comment out this test and accept everything else, I may need some more time to work on the NTM. Shouldn't be long.